game.js

```diff
@@ -0,0 +1,74 @@
+function getComputerChoice(){
```

Comment:
can you try using arrow function Abhishek, that is the more modern way of JS, i would highly recommend